blueoil/utils/config.py

```diff
@@ -44,7 +44,7 @@
     "IMAGE_SIZE",
     "BATCH_SIZE",
     "CLASSES",
-    "PRE_PROCESSOR",
+    ("PRE_PROCESSOR", "TFDS_PRE_PROCESSOR"),
```

Comment:
With this change, can we remove `PRE_PROCESSOR=None` from config files?

Reply:
Yes, as I thought and experimented.